server/services/store/sqlstore/migrations/000019_populate_category_blocks.up.sql

```diff
@@ -1,16 +1,18 @@
-CREATE TABLE {{.prefix}}category_blocks (
+CREATE TABLE {{.prefix}}category_boards (
     id varchar(36) NOT NULL,
     user_id varchar(32) NOT NULL,
```

Comment:
@harshilsharma63 we think this user_id is not needed because the category_id points to the already existing category, that has a user_id. So... can a category be global? looks like this user ID is needed, but I don't have a clear vision of the data model.

Reply:
If it is needed, please make it varchar(36)

Reply:
Okay, so withing storing user_id here as well, handling the query of moving a board from one category to another was getting a little complex. This makes checking for an existing category-block mapping for the current users much easier. I don't remember the exact issue but I remember I intentionally decided to have the user_id here as well.

Reply:
@wiggin77 I've update the user_id length to 36.